src/VisualStudio/CSharp/Test/ProjectSystemShim/LegacyProject/AnalyzersTests.cs

```diff
@@ -243,7 +244,7 @@ public void RuleSet_FileChangingOnDiskRefreshes(bool useCpsProject)
             using var environment = new TestEnvironment();
             if (useCpsProject)
             {
-                CSharpHelpers.CreateCSharpCPSProject(environment, "Test", binOutputPath: null, $"/ruleset:\"{ruleSetFile.Path}\"");
+                await CSharpHelpers.CreateCSharpCPSProjectAsync(environment, "Test", binOutputPath: null, $"/ruleset:\"{ruleSetFile.Path}\"");
```

Comment:
Thanks for moving the tests over to test the non-Obsolete method. We sometimes don't do that and it makes me sad. But this makes me happy. :smile: